### DISCUSSION TITLE
Show clients username in Pro Console

### DISCUSSION THREAD

**Discussion by DMSnz** [rustdesk/rustdesk-server-pro#65] [Ideas]
Just placing this here, I tried to ask this via issues (https://github.com/rustdesk/rustdesk-server-pro/issues/20) and (https://github.com/rustdesk/rustdesk-server-pro/issues/19), but I did not word my request properly I believe so I think its lost in the comments of that old thread.

We got client device name in the console and that was great.
But really wanting client username (e.g. on Windows echo %username%) to show up in the console alongside ID, Device Name etc.
The current username field is seemingly bound to a Pro user login of which I am the one and only so far in our company, where by I intend on having 200+ devices in the console, so client / end point username makes more sense to me.

Cheers :)

**Comment by rustdesk** [comment]
Thanks

**Reply by DMSnz** [reply]
Thanks for inclusion of the latest build.
Can you kindly explain if the user@host should be showing already?
We have updated to 1.1.10 docker and I still only see hostnames in that field?
Would this normally update with API contact from the clients? or only at install time or?
Thanks

**Reply by rustdesk** [reply]
Run RustDesk 1.2.3. `user` info comes from RustDesk 1.2.3 (latest nightly).

**Reply by DMSnz** [reply]
ahhh fail havent deployed that yet, thanks ;)
